docs/source/en/index.mdx

```diff
@@ -12,18 +12,13 @@ specific language governing permissions and limitations under the License.
 
 # 🤗 Transformers
 
-State-of-the-art Machine Learning for PyTorch, TensorFlow and JAX.
+State-of-the-art Transformers models in PyTorch, TensorFlow, and JAX for research and production.
 
-🤗 Transformers provides APIs to easily download and train state-of-the-art pretrained models. Using pretrained models can reduce your compute costs, carbon footprint, and save you time from training a model from scratch. The models can be used across different modalities such as:
+🤗 Transformers is dedicated to providing inference, finetuning, and pretraining for Transformer-based models on text, computer vision, speech, and multimodal tasks. Each model is defined by three classes - a configuration, a model, and a preprocessor - which can be initialized from pretrained instances stored on the [Hugging Face Hub](https://huggingface.co/models). The library also offers convenient helper tools like a simple pipeline to use a model for inference on a given task and a feature-complete training loop optimized for 🤗 Transformers models.
 
-* 📝 Text: text classification, information extraction, question answering, summarization, translation, and text generation in over 100 languages.
-* 🖼️ Images: image classification, object detection, and segmentation.
-* 🗣️ Audio: speech recognition and audio classification.
-* 🐙 Multimodal: table question answering, optical character recognition, information extraction from scanned documents, video classification, and visual question answering.
+Models are available in PyTorch, TensorFlow, and JAX. This framework interoperability grants the flexibility to use a different framework at each stage of a model's lifetime. 🤗 Transformers also supports exporting a model to a format like ONNX and TorchScript for deployment in production environments.
```

Comment:
Adding the support to ONNX and TorchScript is good, but I preferred the deleted sentence which explained more clearly the interoperability.

Reply:
Updated to explain the interoperability a little clearer, let me know what you think :)